.github/workflows/prefect-client.yaml

```diff
@@ -10,6 +12,11 @@ on:
   push:
     branches:
       - main
+    paths:
+      - "**/*.py"
```

Comment:
Do we want to only run on `src/prefect/**.*py` to avoid running on integrations PR?

Reply:
updated in [d5d95e7](https://github.com/PrefectHQ/prefect/pull/12984/commits/d5d95e705ea8e182a69cab51f12982f6af7007dd)